doc/sphinx/proof-engine/ltac.rst

```diff
@@ -129,57 +129,26 @@ The constructs in :token:`ltac_expr` are :term:`left associative`.
 Semantics
 ---------
 
-Tactic expressions can only be applied in the context of a proof. The
-evaluation yields either a term, an integer or a tactic. Intermediate
-results can be terms or integers but the final result must be a tactic
-which is then applied to the focused goals.
+Types of values
+~~~~~~~~~~~~~~~
 
-There is a special case for :tacn:`match goal` expressions of which the clauses
-evaluate to tactics. Such expressions can only be used as the end result of
-a tactic expression (never as an argument of a non-recursive local
-definition or of an application).
+An |Ltac| value can be a tactic, integer, string or syntactic value.
+Syntactic values correspond to certain nonterminal symbols in the grammar,
+each of which is a distinct type of value.
+Most commonly, the value of an |Ltac| expression is a tactic that can be executed.
 
-The rest of this section explains the semantics of each construct of
-|Ltac|.
+While there are a number of constructs that let you combine multiple tactics into
+compound tactics, there are no operations for combining the other types of values.
```

Comment:
Not quite true.  You can combine two `constr`s, two `uconstr`s, two `open_constr`s.  You might be able to combine intropatterns as well, and perhaps some of the others.  But indeed this is true for `string` and `integer`.
```suggestion
compound tactics, there are no operations for combining most other types of values.
```